app/resources/api/v2/pre_capture_pool_resource.rb

```diff
@@ -2,36 +2,34 @@
 
 module Api
   module V2
-    # @todo This documentation does not yet include a detailed description of what this resource represents.
-    # @todo This documentation does not yet include detailed descriptions for relationships, attributes and filters.
-    # @todo This documentation does not yet include any example usage of the API via cURL or similar.
+    # Provides a JSON:API representation of {PreCapturePool}.
+    #
+    # This resource represents a pre-capture pooling process, typically used in sequencing workflows
+    # where samples are combined before the capture step. It provides a unique identifier (`uuid`) for tracking.
```

Comment:
The comment on the model is a little more detailed so could be more useful? Could also link to the 'Submissions in Sequencescape' Confluence page (if we're OK linking to internal Confluence docs in public repo) as that has more information.